engines/grim/lua_v1.cpp

```diff
@@ -497,8 +497,14 @@ void Lua_V1::GetSpeechMode() {
 }
 
 void Lua_V1::GetDiskFreeSpace() {
+	if (g_grim->getGameType() == GType_MONKEY4 &&
+		g_grim->getGamePlatform() == Common::kPlatformPS2) {
+		//wants more than 600 KB
+		lua_pushnumber(700);
+	} else {
 	// amount of free space in MB, used for creating saves
-	lua_pushnumber(50);
+		lua_pushnumber(50);
```

Comment:
This makes me think that an even simpler fix is possible, i.e. just returning 700 for both games. Sure Grim Fandango will then think that you have 700 MB free (who hasn't these days?), but overall, that should fit both.

Then again, this solution makes the behaviour in the two games clear, so it doesn't hurt.